### DISCUSSION TITLE
Could not find my package

### DISCUSSION THREAD

**Discussion by jcs090218** [microsoft/winget-pkgs#188814] [General] [answered]
I've added my package in https://github.com/microsoft/winget-pkgs/pull/155132, but I still couldn't find it in Winget. 🤔 

See https://winget.run/search?query=eask.

Any idea?

**Comment by SpecterShell** [comment, marked as answer]
winget.run is not affiliated to winget, and AFAIK hasn't been updated for a long while.

**Reply by jcs090218** [reply]
Got it! Thanks!
